session/bootstrap.go

```diff
@@ -607,6 +621,14 @@ func upgradeToVer20(s Session) {
 	doReentrantDDL(s, CreateStatsFeedbackTable)
 }
 
+func upgradeToVer21(s Session) {
+	doReentrantDDL(s, "ALTER TABLE mysql.gc_delete_range DROP INDEX element_id", ddl.ErrCantDropFieldOrKey)
+	doReentrantDDL(s, "ALTER TABLE mysql.gc_delete_range DROP INDEX job_id", ddl.ErrCantDropFieldOrKey)
+	doReentrantDDL(s, "ALTER TABLE mysql.gc_delete_range ADD UNIQUE INDEX delete_range_index (job_id, element_id)", ddl.ErrDupKeyName)
```

Comment:
@zimulala Can we add unique index on a table with data?

Reply:
@coocood @tiancaiamao Can we?

Reply:
Yes, we will check every key during adding the unique index.